rmgpy/reduction/reductionTest.py

```diff
@@ -62,9 +64,18 @@ def setUpClass(cls):
         cls.targets = targets
         cls.error = error
 
+        # load kinetic database and forbidden structures
+        rmg.database = RMGDatabase()
+        path = os.path.join(settings['database.directory'])
+
+        # kinetics family loading
+        rmg.database.loadKinetics(os.path.join(path, 'kinetics'),
+                                       kineticsFamilies=['H_Abstraction'],
```

Comment:
Do we need to load `H_Abstraction` in particular for these tests? If not can we get away with loading a smaller family?